### PR TITLE
Fixed broken WordPress galleries CSS

### PR DESCRIPTION
**Pull request by moritzbappert**
### Ticket
- [WordPress galleries layout is wrong](https://app.asana.com/0/484879736575808/1165060274810533/)

### Description
- CSS changes introduced with other features broke the layout of the sliders
- I have now tried to fix it, but some frontend dev should have another look to make sure it looks nice (I don't quite know how it looked before the breaking changed)

### CODE REVIEW

**Comment by moritzbappert** [comment]
This is how it looks for me in GACO:
gaco test_gallerya-test-2

File: assets/styles/style.scss [hunk 1]
```diff
@@ -83,59 +84,66 @@ $color-secondary: #c4c4c4;
...
+        width: 100%;
+        margin: initial;
+        display: block;
+        max-height: 90px;
 
-    &:hover {
-      background: $color-secondary;
+        img {
+          width: auto;
+          display: block;
```
Comment (juanlopez4691) [inline assets/styles/style.scss]:
```suggestion
        img {
          display: block;
          width: auto;
```

File: assets/styles/style.scss [hunk 2]
```diff
@@ -269,12 +277,23 @@ $color-secondary: #c4c4c4;
     }
   }
 
-  .flickity-page-dots {
+  .flickity-prev-next-button {
+    top: auto;
+    width: 28px;
+    height: 28px;
```
Comment (juanlopez4691) [inline assets/styles/style.scss]: Please, add a blank line before the media query.

File: assets/styles/style.scss [hunk 3]
```diff
@@ -269,12 +277,23 @@ $color-secondary: #c4c4c4;
...
+
+  .flickity-page-dots,
+  .flickity-prev-next-button {
     margin: 0;
     padding: 0;
     outline: none;
   }
 
+  .flickity-prev-next-button,
   .flickity-page-dots .dot {
```
Comment (juanlopez4691) [inline assets/styles/style.scss]: For consistency, perhaps we should keep the same selectors order as in lines 289-290.
```suggestion
    .flickity-page-dots .dot,
    .flickity-prev-next-button {
```

**Comment by moritzbappert** [comment]
@juanlopez4691 thanks for your suggestions, I have committed them.

**Pull request merged**